src/arkode/arkode_arkstep.c

```diff
@@ -138,8 +138,10 @@ void* ARKStepCreate(ARKRhsFn fe, ARKRhsFn fi, sunrealtype t0, N_Vector y0,
   ark_mem->step_getnumnonlinsolviters     = arkStep_GetNumNonlinSolvIters;
   ark_mem->step_getnumnonlinsolvconvfails = arkStep_GetNumNonlinSolvConvFails;
   ark_mem->step_getnonlinsolvstats        = arkStep_GetNonlinSolvStats;
+  ark_mem->step_setforcing                = arkStep_SetInnerForcing;
   ark_mem->step_supports_adaptive         = SUNTRUE;
   ark_mem->step_supports_implicit         = SUNTRUE;
+  ark_mem->step_supports_forcing          = SUNTRUE;
```

Comment:
Steven and I collaborated on this piece, since it's used in both this PR and #572.  
@Steven-Roberts, did you need these to be separate, or can I remove the `step_supports_forcing` flag?

Reply:
I see no reason to have the `step_supports_forcing` flag. In fact, I'm already checking if `step_setforcing` is `NULL` rather than using `step_supports_forcing`.